src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -59,6 +60,27 @@ def dcos_browse(name, resource_group_name):
 
     return
 
+def dcos_install_cli(install_location=None):
+    """
+    Downloads the dcos command line from Mesosphere
+    """
+    file_url = ''
+    system = platform.system()
+    if system == 'Windows':
+        file_url = 'https://downloads.dcos.io/binaries/cli/windows/x86-64/dcos-1.8/dcos.exe'
```

Comment:
Suggest to add a command help to call out the version of `1.8`, or expose the version as parameter people can pick; otherwise people will be surprised that we don't install newer stable one after some period.